### DISCUSSION TITLE
Feature: Client implementation of an expert mode button

### DISCUSSION THREAD

**Discussion by Franselbaer** [meshtastic/Discussions#58] [General]
To make it easier for non tech users please implement an expert mode button in all client apps which hides all administrative buttons to give them the ability to simply use and participate on the mesh with (pre)configured devices.

Optionally with a pin?!

**Comment by garthvh** [comment]
This is what managed mode does already.

**Comment by adingbatponder** [comment]
https://meshtastic.org/docs/configuration/radio/security/#managed-mode

**Comment by Franselbaer** [comment]
I know but the buttons shall be hidden, then. This will increase wam factor.

adingbatponder ***@***.***> schrieb am So., 10. Nov. 2024,
10:33:

**Comment by adingbatponder** [comment]
I see what u mean - just wanted to link to previous comment because found it hard to find info on google search. I would find it super if more software in general (irrespective which) would let you chose how to configure menu visibility in a preferences menu. It looks like managed mode is a bit of a hassle to achieve the effect of simpler menu that you would like to see. After all changing setting via pin unlocking on the device itself when in the hand would be easier than total remote control via mesh.

**Comment by Franselbaer** [comment]
Yepp. And secondly we need pocket devices like the t echo without
protruding buttons or antennas or displays like air pod boxes.

adingbatponder ***@***.***> schrieb am So., 10. Nov. 2024,
10:43:

**Comment by Franselbaer** [comment]
And the default channel should be renamed to Meshtastic independent of the
channel settings.

So you get an easy to use out of the box working equipment for non techs....

adingbatponder ***@***.***> schrieb am So., 10. Nov. 2024,
10:43:

**Comment by ikearat** [comment]
Quick question, Managed mode... who would become the Manager?

**Reply by rcarteraz** [reply]
Whoever has remote node administration setup on the device. https://meshtastic.org/docs/configuration/radio/security/#managed-mode
